boards/nxp/mimxrt700_evk/doc/index.rst

```diff
@@ -0,0 +1,192 @@
+.. _mimxrt595_evk:
+
+NXP MIMXRT595-EVK
+##################
+
+Overview
+********
+
+The new i.MX RT700 CPU architecture is composed of a high-performance main compute subsystem,
+a secondary “always-on” sense-compute subsystem and specialized coprocessors.
+
+The main compute subsystem uses a 325 MHz capable Arm® Cortex®-M33 (CM33).
+Similar to the i.MX RT600 crossover MCU, the i.MX RT700 includes a Cadence Tensilica® HiFi 4 DSP.
+The HiFi 4 is a high performance DSP core based upon a Very Long Instruction Word (VLIW) architecture,
+which is capable of processing up to eight 32x16 MACs per instruction cycle. It can be used for offloading
+high-performance numerical tasks such as audio and image processing and supports both fixed-point and
+floating-point operations.
+
+Note: Due to board isn't launched, the picture is blank
```

Comment:
Maybe just don't include an image?  I'm not aware of a requirement that we actually have one.

Reply:
Only release RT700 soc, EVK board don't launch currently. We can't show the board picture currently.